src/hooks/usePublicMessages.js

```diff
@@ -48,13 +50,13 @@ export default function usePublicMessages(infoPoint, routes) {
   return useMemo(() => {
     if (!(publicMessages instanceof Array)) return publicMessages;
     return publicMessages.sort(comparePublicMessages).filter((publicMessage) => {
-      if ((routes instanceof Array) && (publicMessage.routeAbbreviation)) {
-        return routes.includes(publicMessage.routeAbbreviation);
+      if (routes instanceof Array && publicMessage.routes) {
```

Comment:
```suggestion
      if (routes instanceof Array) {
```
With your updates (at least according to your docs), `publicMessage.routes` will always be an array, so the second part always is truthy.

Reply:
Now that you mention it, it would make more sense for `publicMessage.routes` to be null if no routes are tied to it, that would flag it as a general message and help avoid using `'ALL'` in the hook code

Reply:
Indeed, that might help make a more elegant solution to the filtering problems I outlined below as well